test/images/image-util.sh

```diff
@@ -145,11 +148,14 @@ build() {
         ${SED} -i "s|QEMUARCH|${QEMUARCHS[$arch]}|g" Dockerfile
         # Register qemu-*-static for all supported processors except the current one
         echo "Registering qemu-*-static binaries in the kernel"
-        local sudo=""
-        if [[ $(id -u) != 0 ]]; then
-          sudo=sudo
-        fi
-        ${sudo} "${KUBE_ROOT}/third_party/multiarch/qemu-user-static/register/register.sh" --reset
+
+        # NOTE(claudiub): In order to build multiarch images with docker buildx, we need qemu and binfmt-support
+        # installed. They might not be installed or available for this OS (for example the image
+        # gcr.io/k8s-testimages/gcb-docker-gcloud does not have them, nor are there APKs available for
+        # binfmt-support).
+        # The multiarch/qemu-user-static image already has everything we need.
+	# https://medium.com/@artur.klauser/building-multi-architecture-docker-images-with-buildx-27d80f7e2408
+        docker run --rm --privileged multiarch/qemu-user-static:5.1.0-2 --reset -p yes
```

Comment:
is it possible to use an immutable hash instead of a tag? a malicious user could overwrite the `5.1.0-2` tag, resulting in developers running unknown and unaudited code as root.

Reply:
alternately (or maybe in addition), can we mirror this image on k8s.gcr.io? at least then we'd have some additional assurance over the provenance of this image.

Reply:
I've put an immutable hash.

Reply:
+1 to mirroring might also be a good idea, we have other projects that need to do this sort of thing (e.g. kind)

Reply:
Indeed, it might be a good idea since a lot of image builders are using this image to build multiarch images.